README.md

```diff
@@ -1,50 +1,74 @@
+
+*May* means that an item is truly optional. It's an individual decision for each addon within the org to implement this item or not. A given decision should not be questioned to avoid changing it back and forth. 
+
+> This terminology is based on [RFC 2119](https://tools.ietf.org/html/rfc2119), which is used by many specification documents.
+
+### Changelog
+
+All addons within the org should have a changelog. The changelog may not cover versions that were released before it was introduced.
+
+> Changelogs are important to communicate end users what is going on with the project. See [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) for additional reasoning.
+
+The changelog should be generated and updated automatically as part of the release process. [`lerna-changelog`](https://github.com/lerna/lerna-changelog) should be used to do so.
+
+> Lerna-changelog generates changelogs based on GitHub pull requests and their labels. Please refer to [it's documentation](https://github.com/lerna/lerna-changelog#lerna-changelog) for more details.
+
+lerna-changelog can be integrated with `release-it` to automatically generate the changelog as part of the release process. Robert Jackson (`@rwjblue`) provides a script to automate the setup for these tools: [create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup). It should be used for the setup and as a reference for recommended configuration.
+
+> Please refer to the [documentation of create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup#create-rwjblue-release-it-setup) for usage instructions. The script could be rerun on a repository already using lerna-changelog and release-it to update the configuration to the latest recommendations.
 
-* If your addon does not have a changelog, consider adding one
-	- Changelogs are important to communicate to end users what is going on with the project. See [keepachangelog.com](https://keepachangelog.com/en/1.0.0/) for additional reasoning.
-	- Robert Jackson (@rwjblue) has automated the initial setup of automatic changelog generation using some popular tools. Follow the steps in [create-rwjblue-release-it-setup](https://github.com/rwjblue/create-rwjblue-release-it-setup). You can see an example of how that works in [ember-pikaday](https://github.com/adopted-ember-addons/ember-pikaday).
+### GitHub labels
 
-* Use standard labels for issues and pull requests.
-	* Defer to the learning team's recommendations and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
+GitHub issues and pull request should be labeled. It should follow the [learning team's recommendations](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#issue-labeling) and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
```

Comment:
I wasn't clear what *learning team's recommendations* refers to. I only found the chapter in contributing docs of `ember-source` repo. But that refers to a third-party document. The text and link doesn't fit well together anymore. Are there any documented recommendations from the learning team that I missed? Otherwise I suggest that we reword:

```suggestion
GitHub issues and pull request should be labeled. It should follow the [rules for ember-source repository](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#issue-labeling) and [lerna-changelog](https://github.com/lerna/lerna-changelog)'s defaults.
```

Reply:
This was my ambiguous thought! I was thinking that the learning team as one of the ember core teams spends a lot of time thinking about how to best present topics and manage contributions. That said, it doesn't necessarily follow that they've built a taxonomy for their labels. So I intended to say "whatever ember core project that has a well defined issue labeling convention". I'd hope for us to share work since the OSS dimension to addons seems quite similar to the core projects.